tracer/src/Datadog.Trace/Activity/Handlers/IgnoreActivityHandler.cs

```diff
@@ -18,6 +18,7 @@ internal class IgnoreActivityHandler : IActivityHandler
         private static readonly string[] SourcesNames =
         {
             "Couchbase.DotnetSdk.RequestTracer",
+            "Grpc.Net.Client",
```

Comment:
I'm not 100% sure whether this one is necessary - the activities are ignored via the `IgnoreOperationNamesStartingWith`

I thought the source name would be `Grpc.Net.Client`, but there doesn't seem to be one? [code](https://github.com/grpc/grpc-dotnet/blob/6bf44820a2ef6af5845830728cd4b3d1a8053ed4/src/Grpc.Net.Client/Internal/GrpcDiagnostics.cs#L26)